### PR TITLE
Updated Documentation for EntraBeta 

### PR DESCRIPTION
**Pull request by snehalkotwal**

### CODE REVIEW

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Add-EntraBetaGroupMember.md [hunk 1]
```diff
@@ -30,23 +31,28 @@ Add-EntraBetaGroupMember
 ```
 
 ## Description
-The Add-EntraBetaGroupMember cmdlet adds a member to a group.
+
+The `Add-EntraBetaGroupMember` cmdlet adds a member to a group. Specify the `ObjectId` and `RefObjectId` parameters to add a member to a group.
```
Comment (KenitoInc) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Add-EntraBetaGroupMember.md:35]: You could explain what `ObjectId` and `RefObjectId` are similar to what was done [here](https://github.com/microsoftgraph/entra-powershell/blob/main/module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Add-EntraDeviceRegisteredOwner.md#example-1)

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Add-EntraBetaGroupOwner.md [hunk 2]
```diff
@@ -8,35 +18,41 @@ schema: 2.0.0
...
+Add-EntraBetaGroupOwner 
+ -ObjectId <String>
+ -RefObjectId <String>
+ [<CommonParameters>]
 ```
 
 ## Description
-The Add-EntraBetaGroupOwner cmdlet adds an owner to an Azure Active Directory group.
+
+The `Add-EntraBetaGroupOwner` cmdlet adds an owner to a Microsoft Entra ID group. Specify the `ObjectId` and `RefObjectId` parameters to add an owner to a group.
```
Comment (KenitoInc) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Add-EntraBetaGroupOwner.md:35]: You could explain what `ObjectId` and `RefObjectId` are similar to what was done [here](https://github.com/microsoftgraph/entra-powershell/blob/main/module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Add-EntraDeviceRegisteredOwner.md#example-1)

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Add-EntraBetaGroupMember.md (hunk 1 above)
Comment (snehalkotwal) [reply module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Add-EntraBetaGroupMember.md:35]: File updated.

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Add-EntraBetaGroupOwner.md (hunk 2 above)
Comment (snehalkotwal) [reply module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Add-EntraBetaGroupOwner.md:35]: File updated.

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Add-EntraBetaGroupMember.md [hunk 3]
```diff
@@ -30,23 +31,32 @@ Add-EntraBetaGroupMember
...
+`-RefObjectId` - specifies the unique identifier (Object ID) of the member to be added to the group.
 
 ## Examples
 
 ### Example 1: Add a member to a group
+
 ```powershell
-PS C:\> Add-EntraBetaGroupMember -ObjectId "056b2531-005e-4f3e-be78-01a71ea30a04" -RefObjectId "996d39aa-fdac-4d97-aa3d-c81fb47362ac"
+Connect-Entra -Scopes 'GroupMember.ReadWrite.All'
+Add-EntraBetaGroupMember -ObjectId 'dddddddd-2222-3333-5555-rrrrrrrrrrrr' -RefObjectId 'bbbbbbbb-1111-2222-3333-cccccccccccc'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Add-EntraBetaGroupMember.md:47]:
```suggestion
$params = @{
    ObjectId = 'dddddddd-2222-3333-5555-rrrrrrrrrrrr'
    RefObjectId = 'bbbbbbbb-1111-2222-3333-cccccccccccc'
}

Add-EntraBetaGroupMember @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Add-EntraBetaGroupOwner.md [hunk 4]
```diff
@@ -8,35 +18,45 @@ schema: 2.0.0
...
 
 ## Examples
 
 ### Example 1: Add an owner to a group
-```
-PS C:\>Add-EntraBetaGroupOwner -ObjectId "62438306-7c37-4638-a72d-0ee8d9217680" -RefObjectId "0a1068c0-dbb6-4537-9db3-b48f3e31dd76"
+
+```powershell
+Connect-Entra -Scopes 'Group.ReadWrite.All'
+Add-EntraBetaGroupOwner -ObjectId 'hhhhhhhh-3333-5555-3333-qqqqqqqqqqqq' -RefObjectId 'bbbbbbbb-1111-2222-3333-cccccccccccc'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Add-EntraBetaGroupOwner.md:47]:
```suggestion
$params = @{
    ObjectId = 'hhhhhhhh-3333-5555-3333-qqqqqqqqqqqq'
    RefObjectId = 'bbbbbbbb-1111-2222-3333-cccccccccccc'
}

Add-EntraBetaGroupOwner @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaGroup.md [hunk 5]
```diff
@@ -8,90 +18,170 @@ schema: 2.0.0
...
 
 ## Examples
 
-### Example 1: Create a dynamic group
+### Example 1: Create a group
+
+```powershell
+Connect-Entra -Scopes 'Group.ReadWrite.All' #Delegated Permission
+Connect-Entra -Scopes 'Group.Create' #Application permission
+New-EntraBetaGroup -DisplayName 'HelpDesk admin group2' -MailEnabled $False -MailNickname 'helpDeskAdminGroup' -SecurityEnabled $True
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaGroup.md:60]:
```suggestion
$params = @{
    DisplayName = 'HelpDesk admin group2'
    MailEnabled = $False
    MailNickname = 'helpDeskAdminGroup'
    SecurityEnabled = $True
}

New-EntraBetaGroup @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaGroup.md [hunk 6]
```diff
@@ -8,90 +18,170 @@ schema: 2.0.0
...
+This example demonstrates how to create the new group.
 
-\`user.department -contains "Marketing"\`
+### Example 2: Create a group with Description parameter
 
-The double quotation marks are replaced with single quotation marks.
+```powershell
+Connect-Entra -Scopes 'Group.ReadWrite.All' #Delegated Permission
+Connect-Entra -Scopes 'Group.Create' #Application permission
+New-EntraBetaGroup -DisplayName 'HelpDesk admin group' -MailEnabled $false -MailNickname 'helpDeskAdminGroup' -SecurityEnabled $true  -Description 'Group assignable to role'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaGroup.md:76]:
```suggestion
$params = @{
    DisplayName = 'HelpDesk admin group'
    MailEnabled = $false
    MailNickname = 'helpDeskAdminGroup'
    SecurityEnabled = $true
    Description = 'Group assignable to role'
}

New-EntraBetaGroup @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaGroup.md [hunk 7]
```diff
@@ -8,90 +18,170 @@ schema: 2.0.0
...
+```
+
+This example demonstrates how to create the new group with description parameter.
+
+### Example 3: Create a group with IsAssignableToRole parameter
+
+```powershell
+Connect-Entra -Scopes 'Group.ReadWrite.All' #Delegated Permission
+Connect-Entra -Scopes 'Group.Create' #Application permission
+New-EntraBetaGroup -DisplayName 'HelpDesk admin group2' -Description 'Group assignable to role' -MailEnabled $False -MailNickname 'helpDeskAdminGroup' -SecurityEnabled $True -IsAssignableToRole $True
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaGroup.md:94]:
```suggestion
$params = @{
    DisplayName = 'HelpDesk admin group2'
    Description = 'Group assignable to role'
    MailEnabled = $False
    MailNickname = 'helpDeskAdminGroup'
    SecurityEnabled = $True
    IsAssignableToRole = $True
}

New-EntraBetaGroup @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaGroup.md [hunk 8]
```diff
@@ -8,90 +18,170 @@ schema: 2.0.0
...
-GroupTypes                    : {}
+
+This example demonstrates how to create the new group with IsAssignableToRole parameter.
+
+### Example 4: Create a group with Visibility parameter
+
+```powershell
+Connect-Entra -Scopes 'Group.ReadWrite.All' #Delegated Permission
+Connect-Entra -Scopes 'Group.Create' #Application Permission
+New-EntraBetaGroup -DisplayName 'HelpDesk admin group2' -Description 'Group assignable to role' -MailEnabled $False -MailNickname 'helpDeskAdminGroup' -SecurityEnabled $True -Visibility 'Private'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaGroup.md:110]:
```suggestion
$params = @{
    DisplayName = 'HelpDesk admin group2'
    Description = 'Group assignable to role'
    MailEnabled = $False
    MailNickname = 'helpDeskAdminGroup'
    SecurityEnabled = $True
    Visibility = 'Private'
}

New-EntraBetaGroup @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaGroup.md [hunk 9]
```diff
@@ -8,90 +18,170 @@ schema: 2.0.0
...
+```
+
+This example demonstrates how to create the new group with Visibility parameter.
+
+### Example 5: Create a group with GroupTypes parameter
+
+```powershell
+Connect-Entra -Scopes 'Group.ReadWrite.All' #Delegated Permission
+Connect-Entra -Scopes 'Group.Create' #Application permission
+New-EntraBetaGroup -DisplayName 'HelpDesk admin group3' -Description 'group des' -MailEnabled $False -MailNickname 'helpDeskAdminGroup1' -SecurityEnabled $True -GroupTypes 'Unified'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaGroup.md:126]:
```suggestion
$params = @{
    DisplayName = 'HelpDesk admin group3'
    Description = 'group des'
    MailEnabled = $False
    MailNickname = 'helpDeskAdminGroup1'
    SecurityEnabled = $True
    GroupTypes = 'Unified'
}

New-EntraBetaGroup @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaGroup.md [hunk 10]
```diff
@@ -8,90 +18,170 @@ schema: 2.0.0
...
+```
+
+This example demonstrates how to create the new group with GroupTypes parameter.
+
+### Example 6: Create a group membership rule processing state parameter
+
+```powershell
+Connect-Entra -Scopes 'Group.ReadWrite.All' #Delegated Permission
+Connect-Entra -Scopes 'Group.Create' #Application permission
+New-EntraBetaGroup -DisplayName 'HelpDesk admin group2'  -MailEnabled $False -MailNickname 'helpDeskAdminGroup' -SecurityEnabled $True  -MembershipRuleProcessingState 'On'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaGroup.md:142]:
```suggestion
$params = @{
    DisplayName = 'HelpDesk admin group2'
    MailEnabled = $False
    MailNickname = 'helpDeskAdminGroup'
    SecurityEnabled = $True
    MembershipRuleProcessingState = 'On'
}

New-EntraBetaGroup @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaGroup.md [hunk 11]
```diff
@@ -8,90 +18,170 @@ schema: 2.0.0
...
+```
+
+This example demonstrates how to create the new group with MembershipRuleProcessingState parameter
+
+### Example 7: Create a group membership rule parameter
+
+```powershell
+Connect-Entra -Scopes 'Group.ReadWrite.All' #Delegated Permission
+Connect-Entra -Scopes 'Group.Create' #Application permission
+New-EntraBetaGroup -DisplayName 'HelpDesk admin group2'  -MailEnabled $False -MailNickname 'helpDeskAdminGroup' -SecurityEnabled $True -MembershipRule '(user.department -contains "Marketing")' -MembershipRuleProcessingState 'On'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaGroup.md:158]:
```suggestion
$params = @{
    DisplayName = 'HelpDesk admin group2'
    MailEnabled = $False
    MailNickname = 'helpDeskAdminGroup'
    SecurityEnabled = $True
    MembershipRule = '(user.department -contains "Marketing")'
    MembershipRuleProcessingState = 'On'
}

New-EntraBetaGroup @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaGroup.md [hunk 12]
```diff
@@ -200,12 +280,13 @@ Accept wildcard characters: False
 ```
 
 ### -MembershipRule
+
 Specifies the membership rule for a dynamic group.
 
-For more information about the rules that you can use for dynamic groups, see Using attributes to create advanced rules (https://azure.microsoft.com/en-us/documentation/articles/active-directory-accessmanagement-groups-with-advanced-rules/).
+For more information about the rules that you can use for dynamic groups, see Using attributes to create advanced rules (https://azure.microsoft.com/documentation/articles/active-directory-accessmanagement-groups-with-advanced-rules/).
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaGroup.md:286]:
```suggestion
For more information about the rules that you can use for dynamic groups, see Using attributes to create advanced rules (https://learn.microsoft.com/entra/identity/users/groups-dynamic-membership).
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaGroup.md [hunk 13]
```diff
@@ -282,25 +366,29 @@ Accept wildcard characters: False
...
+[Get-EntraBetaGroup](Get-EntraBetaGroup.md)
 
-[Remove-EntraBetaGroup]()
+[Remove-EntraBetaGroup](Remove-EntraBetaGroup.md)
 
-[Set-EntraBetaGroup]()
+[Set-EntraBetaGroup](Set-EntraBetaGroup.md)
 
-[Using attributes to create advanced rules](https://azure.microsoft.com/en-us/documentation/articles/active-directory-accessmanagement-groups-with-advanced-rules/)
+[Using attributes to create advanced rules](https://azure.microsoft.com/documentation/articles/active-directory-accessmanagement-groups-with-advanced-rules/)
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaGroup.md:394]:
```suggestion
[Using attributes to create advanced rules](https://learn.microsoft.com/entra/identity/users/groups-dynamic-membership)
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Remove-EntraBetaGroupMember.md [hunk 14]
```diff
@@ -18,43 +18,41 @@ schema: 2.0.0
...
 ## Examples
 
 ### Example 1: Remove a member
 
-This example demonstrates how to remove a member from a group in Microsoft Entra ID.
-
 ```powershell
-PS C:\>Remove-EntraBetaGroupMember -ObjectId "62438306-7c37-4638-a72d-0ee8d9217680" -MemberId "0a1068c0-dbb6-4537-9db3-b48f3e31dd76"
+Connect-Entra -Scopes 'GroupMember.ReadWrite.All'
+Remove-EntraBetaGroupMember -ObjectId 'hhhhhhhh-3333-5555-3333-qqqqqqqqqqqq' -MemberId 'zzzzzzzz-6666-8888-9999-pppppppppppp'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Remove-EntraBetaGroupMember.md:43]:
```suggestion
$params = @{
    ObjectId = 'hhhhhhhh-3333-5555-3333-qqqqqqqqqqqq'
    MemberId = 'zzzzzzzz-6666-8888-9999-pppppppppppp'
}

Remove-EntraBetaGroupMember @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Remove-EntraBetaGroupOwner.md [hunk 15]
```diff
@@ -8,32 +18,41 @@ schema: 2.0.0
...
 
 ## Examples
 
 ### Example 1: Remove an owner
-```
-PS C:\>Remove-EntraBetaGroupOwner -ObjectId "62438306-7c37-4638-a72d-0ee8d9217680" -OwnerId "0a1068c0-dbb6-4537-9db3-b48f3e31dd76"
+
+```powershell
+Connect-Entra -Scopes 'Group.ReadWrite.All'
+Remove-EntraBetaGroupOwner -ObjectId 'qqqqqqqq-5555-0000-1111-hhhhhhhhhhhh' -OwnerId 'xxxxxxxx-8888-5555-9999-bbbbbbbbbbbb'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Remove-EntraBetaGroupOwner.md:43]:
```suggestion
$params = @{
    ObjectId = 'qqqqqqqq-5555-0000-1111-hhhhhhhhhhhh'
    OwnerId = 'xxxxxxxx-8888-5555-9999-bbbbbbbbbbbb'
}

Remove-EntraBetaGroupOwner @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaGroup.md [hunk 16]
```diff
@@ -8,72 +18,113 @@ schema: 2.0.0
...
+The `Set-EntraBetaGroup` cmdlet sets the properties for an existing Microsoft Entra ID group. Specify the `Id` parameter to set the properties for an existing Microsoft Entra ID group.
 
 ## Examples
 
-### Example 1
+### Example 1: Update a group display name
+
+```powershell
+Connect-Entra -Scopes 'Group.ReadWrite.All'
+Set-EntraBetaGroup -Id 'kkkkkkkk-3333-5555-1111-nnnnnnnnnnnn' -DisplayName 'UPDATE helpdesk'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaGroup.md:52]:
```suggestion
$params = @{
    Id = 'kkkkkkkk-3333-5555-1111-nnnnnnnnnnnn'
    DisplayName = 'UPDATE helpdesk'
}

Set-EntraBetaGroup @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaGroup.md [hunk 17]
```diff
@@ -8,72 +18,113 @@ schema: 2.0.0
...
-GroupTypes                    : {} 
-MembershipRule                : (user.department -eq "Marketing") MembershipRuleProcessingState : Paused
+
+This command updates the display name of a specified group in Microsoft Entra ID.
+
+### Example 2: Update a group description
+
+```powershell
+Connect-Entra -Scopes 'Group.ReadWrite.All'
+Set-EntraBetaGroup -Id 'kkkkkkkk-3333-5555-1111-nnnnnnnnnnnn' -Description 'This is my new group'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaGroup.md:61]:
```suggestion
$params = @{
    Id = 'kkkkkkkk-3333-5555-1111-nnnnnnnnnnnn'
    Description = 'This is my new group'
}

Set-EntraBetaGroup @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaGroup.md [hunk 18]
```diff
@@ -8,72 +18,113 @@ schema: 2.0.0
...
 
-Group updated.
+This example demonstrates how to update a group description.  
 
-### Example 2
+### Example 3: Update a group mail nickname
+
+```powershell
+Connect-Entra -Scopes 'Group.ReadWrite.All'
+Set-EntraBetaGroup -Id 'kkkkkkkk-3333-5555-1111-nnnnnnnnnnnn' -MailNickName 'newnickname'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaGroup.md:70]:
```suggestion
$params = @{
    Id = 'kkkkkkkk-3333-5555-1111-nnnnnnnnnnnn'
    MailNickName = 'newnickname'
}

Set-EntraBetaGroup @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaGroup.md [hunk 19]
```diff
@@ -8,72 +18,113 @@ schema: 2.0.0
...
 ```
-PS C:\> Set-EntraBetaGroup -Id "11111111-1111-1111-1111-111111111111" -LabelId "00000000-0000-0000-0000-000000000000"
+
+This example demonstrates how to update a group main enabled.  
+
+### Example 6: Update a property for a group
+
+```powershell
+Connect-Entra -Scopes 'Group.ReadWrite.All'
+Set-EntraBetaGroup -Id 'kkkkkkkk-3333-5555-1111-nnnnnnnnnnnn' -Visibility 'Private' -GroupTypes 'DynamicMembership' -IsAssignableToRole $true
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaGroup.md:97]:
```suggestion
$params = @{
    Id = 'kkkkkkkk-3333-5555-1111-nnnnnnnnnnnn'
    Visibility = 'Private'
    GroupTypes = 'DynamicMembership'
    IsAssignableToRole = $true
}

Set-EntraBetaGroup @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaGroup.md [hunk 20]
```diff
@@ -8,72 +18,113 @@ schema: 2.0.0
...
-PS C:\> Set-EntraBetaGroup -Id "9126185e-25df-4522-a380-7ab697a7241c" -IsAssignableToRole $true
-        Bad Request.
+
+This command updates the mail nickname of a specified group in Microsoft Entra ID.
+
+### Example 4: Update a group security enabled
+
+```powershell
+Connect-Entra -Scopes 'Group.ReadWrite.All'
+Set-EntraBetaGroup -Id 'kkkkkkkk-3333-5555-1111-nnnnnnnnnnnn' -SecurityEnabled $true
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaGroup.md:79]:
```suggestion
$params = @{
    Id = 'kkkkkkkk-3333-5555-1111-nnnnnnnnnnnn'
    SecurityEnabled = $true
}

Set-EntraBetaGroup @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaGroup.md [hunk 21]
```diff
@@ -8,72 +18,113 @@ schema: 2.0.0
...
+### Example 3: Update a group mail nickname
+
+```powershell
+Connect-Entra -Scopes 'Group.ReadWrite.All'
+Set-EntraBetaGroup -Id 'kkkkkkkk-3333-5555-1111-nnnnnnnnnnnn' -MailNickName 'newnickname'
 ```
-PS C:\> Set-EntraBetaGroup -Id "9126185e-25df-4522-a380-7ab697a7241c" -IsAssignableToRole $true
-        Bad Request.
+
+This command updates the mail nickname of a specified group in Microsoft Entra ID.
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaGroup.md:73]:
```suggestion
This example shows how to update a group's mail nickname in Microsoft Entra ID.
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaGroup.md [hunk 22]
```diff
@@ -8,72 +18,113 @@ schema: 2.0.0
...
+
+### Example 4: Update a group security enabled
+
+```powershell
+Connect-Entra -Scopes 'Group.ReadWrite.All'
+Set-EntraBetaGroup -Id 'kkkkkkkk-3333-5555-1111-nnnnnnnnnnnn' -SecurityEnabled $true
 ```
 
-IsassignableToRole property cannot be set for an existing group.
+This command updates the security enabled of a specified group in Microsoft Entra ID.
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaGroup.md:82]:
```suggestion
This example shows how to update a group's security settings in Microsoft Entra ID.
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaGroup.md [hunk 23]
```diff
@@ -8,72 +18,113 @@ schema: 2.0.0
...
 
-The label is assigned to the group.
+This example demonstrates how to update a property for an existing Microsoft Entra ID group.  
+
+### Example 7: Update a group membership rule
 
-### Example 4
+```powershell
+Connect-Entra -Scopes 'Group.ReadWrite.All'
+Set-EntraBetaGroup -Id 'kkkkkkkk-3333-5555-1111-nnnnnnnnnnnn' -MembershipRule '(user.UserType -contains "Member")'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaGroup.md:106]:
```suggestion
$params = @{
    Id = 'kkkkkkkk-3333-5555-1111-nnnnnnnnnnnn'
    MembershipRule = '(user.UserType -contains "Member")'
}

Set-EntraBetaGroup @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaGroup.md [hunk 24]
```diff
@@ -8,72 +18,113 @@ schema: 2.0.0
...
 ```
-PS C:\> Set-EntraBetaGroup -Id "11111111-1111-1111-1111-111111111111" -LabelId ""
+
+This example demonstrates how to update the membership rule of a specified group in Microsoft Entra ID.
+
+### Example 8: Update a group membership rule processing state
+
+```powershell
+Connect-Entra -Scopes 'Group.ReadWrite.All'
+Set-EntraBetaGroup -Id 'kkkkkkkk-3333-5555-1111-nnnnnnnnnnnn' -MembershipRuleProcessingState 'On'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaGroup.md:115]:
```suggestion
$params = @{
    Id = 'kkkkkkkk-3333-5555-1111-nnnnnnnnnnnn'
    MembershipRuleProcessingState = 'On'
}

Set-EntraBetaGroup @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaUser.md [hunk 25]
```diff
@@ -8,48 +18,112 @@ schema: 2.0.0
...
 ```
 
-This command updates the specified user's property.
+This example updates the specified user's as minors with parental consent.
+
+### Example 4: Set the specified user's parameter
+
+```powershell
+Connect-Entra -Scopes 'User.ReadWrite.All'
+Set-EntraBetaUser -ObjectId 'aaaaaaaa-6666-7777-8888-bbbbbbbbbbbb' -City 'Add city name' -CompanyName 'Microsoft' -ConsentProvidedForMinor 'Granted' -Country 'Add country name' -Department 'Add department name' -GivenName 'Mircosoft' -ImmutableId '#1' -JobTitle 'Manager' -MailNickName 'Add mailnickname' -Mobile '9984534564' -OtherMails 'test12@M365x99297270.OnMicrosoft.com' -PasswordPolicies 'DisableStrongPassword' -State 'UP' -StreetAddress 'Add address' -UserType 'Member'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaUser.md:100]:
```suggestion
$params = @{
    ObjectId = 'aaaaaaaa-6666-7777-8888-bbbbbbbbbbbb'
    City = 'Add city name'
    CompanyName = 'Microsoft'
    ConsentProvidedForMinor = 'Granted'
    Country = 'Add country name'
    Department = 'Add department name'
    GivenName = 'Mircosoft'
    ImmutableId = '#1'
    JobTitle = 'Manager'
    MailNickName = 'Add mailnickname'
    Mobile = '9984534564'
    OtherMails = 'test12@M365x99297270.OnMicrosoft.com'
    PasswordPolicies = 'DisableStrongPassword'
    State = 'UP'
    StreetAddress = 'Add address'
    UserType = 'Member'
}

Set-EntraBetaUser @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaUser.md [hunk 26]
```diff
@@ -139,7 +218,8 @@ Accept wildcard characters: False
 ```
 
 ### -ExtensionProperty
-@{Text=}
+
+Add data to custom user properties as the basic open extensions or the more versatile schema extensions. See [more about extensions][Learn more about extensions].
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaUser.md:222]:
```suggestion
Add data to custom user properties as the basic open extensions or the more versatile schema extensions. 
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaUser.md [hunk 27]
```diff
@@ -499,10 +554,11 @@ Accept wildcard characters: False
 ```
 
 ### -AgeGroup
-{{ Fill AgeGroup Description }}
+
+Used by enterprise applications to determine the legal age group of the user. This property is read-only and calculated based on ageGroup and consentProvidedForMinor properties. Allowed values: null, minor, notAdult, and adult. Refer to the [legal age group property definitions][Learn more about age group and minor consent definitions].
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaUser.md:558]:
```suggestion
Used by enterprise applications to determine the legal age group of the user. This property is read-only and calculated based on ageGroup and consentProvidedForMinor properties. Allowed values: null, minor, notAdult, and adult. 
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaUserManager.md [hunk 28]
```diff
@@ -8,33 +18,41 @@ schema: 2.0.0
...
 
 ## Examples
 
 ### Example 1: Update a user's manager
-```
-PS C:\>Set-EntraBetaUserManager -ObjectId "df19e8e6-2ad7-453e-87f5-037f6529ae16" -RefObjectId "df19e8e6-2ad7-453e-87f5-037f6529ae16"
+
+```powershell
+Connect-Entra -Scopes 'User.ReadWrite.All'
+Set-EntraBetaUserManager -ObjectId 'bbbbbbbb-1111-2222-3333-cccccccccccc' -RefObjectId '55ff55ff-aa66-bb77-cc88-99dd99dd99dd'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaUserManager.md:43]:
```suggestion
$params = @{
    ObjectId = 'bbbbbbbb-1111-2222-3333-cccccccccccc'
    RefObjectId = '55ff55ff-aa66-bb77-cc88-99dd99dd99dd'
}

Set-EntraBetaUserManager @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Remove-EntraBetaUserManager.md [hunk 29]
```diff
@@ -8,38 +18,43 @@ schema: 2.0.0
...
+```powershell
+Connect-Entra -Scopes 'User.ReadWrite.All'
+$User = Get-EntraBetaUser -Top 1
+Remove-EntraBetaUserManager -ObjectId $User.ObjectId
 ```
 
-The first command gets a user by using the Get-EntraBetaUser (./Get-EntraBetaUser.md)cmdlet, and then stores it in the $User variable.
+The first command gets a user by using the [Get-EntraBetaUser](./Get-EntraBetaUser.md) cmdlet, and then stores it in the $User variable.
 
 The second command removes the user in $User.
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Remove-EntraBetaUserManager.md:48]:
```suggestion
This example shows how to remove a user's manager.

You can use `Get-EntraUser` command to get the user's details.
```

**Review by SteveMutungi254 — APPROVED** [review]
LGTM.

**Pull request merged**